.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+yarn.lock
```

Comment:
We don't use Yarn in this repo so shouldn't really need this =/ This seems like it should be in the developer's global `.gitignore` -- or they should use `npm install`

Will leave it though I guess, doesn't really hurt anyone, though might confuse them